delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/plugin/PluginsClassLoader.scala

```diff
@@ -20,7 +26,7 @@ class PluginsClassLoader(pluginsClassLoader: List[PluginClassLoader], parent: Cl
     */
   override def loadClass(className: String): Class[_] =
     loadClassFromParent(className)
-      .orElse(loadClassFromPlugins(className))
+      .orElse(loadClassFromPlugins(className, cls.toList))
```

Comment:
why don't you use directly the ListBuffer ?
It doesn't seem efficient to me to convert the buffer to a list every time a class is attempted to be loaded from the plugins.

Reply:
To avoid later possible concurrent modification errors. The number of entries is very low, so there should be no problems.